index.bs

```diff
@@ -736,28 +881,15 @@ Reflective modules are always already instantiated.
 1. Return _evaluate_().
 </emu-alg>
 
-<h4 id="parse-exports-descriptors" aoid="ParseExportsDescriptors">ParseExportsDescriptors(obj)</h4>
+<h3 id="module-constructor">The Reflect.Module Constructor</h3>
 
-<b>TODO:</b> parse as in <a href="https://gist.github.com/dherman/fbf3077a2781df74b6d8">these examples</a>
-<ul>
-  <li>uninitialized, mutable: <code>{ }</code>
-  <li>uninitialized, immutable: <code>{ const: true }</code>
-  <li>initialized, mutable: <code>{ value: 42 }</code>
-  <li>initialized, immutable: <code>{ value: 42, const: true }</code>
-  <li>re-export (immutable): <code>{ module: m, import: "foo" }</code>
-</ul>
-
-<emu-alg>
-1. // TODO: spec me
-</emu-alg>
+The Module constructor is the initial value of the Module property of the the Reflect object. When called as a constructor it creates and initializes a new Module object. Reflect.Module is not intended to be called as a function and will throw an exception when called in that manner.
 
-<h4 id="create-module-mutator" aoid="CreateModuleMutator">CreateModuleMutator(module)</h4>
+The Reflect.Module constructor is designed to be subclassable. It may be used as the value in an extends clause of a class definition. Subclass constructors that intend to inherit the specified Module behaviour must include a super call to the Reflect.Module constructor to create and initialize the subclass instance with the internal state necessary to support the Reflect.Module.prototype built-in methods.
```

Comment:
change the end of this to "with the internal state necessary to integrated with loaders."